### ISSUE TITLE
Cost Function org.openda.algorithms.SimulationKwadraticCostFunction not accessable anymore

### ISSUE THREAD

**Issue by theavuik** [OpenDA-Association/OpenDA#320]
Schema specifies org.openda.algorithms.SimulationKwadraticCostFunction and in code it starts with default value org.openda.algorithms.SimulationKwadraticCostFunction but this is automatically translated to org.openda.algorithms.SimulationKwadraticCostFunctionWithGradient

For BFGS & ConjugateGradient it is like this, but for others, it is available like Delsa and BaseDud.

This issue is a duplicate of JIRA issue ODA-548, which will now be closed.

Earlier comments on this issue: 
![afbeelding](https://github.com/OpenDA-Association/OpenDA/assets/7456695/cb844de4-830a-4ed2-bad5-e74631af0044)

**Comment by theavuik** [comment]
@erikpelgrim is this still an issue?

**Comment by erikpelgrim** [comment]
Yes, good to pick up the next time something is done for cost functions
